src/SystemCommands-MessageCommands/SycRemoveAllSendersCommand.class.st

```diff
@@ -1,5 +1,5 @@
 "
-I am a command to launch the Remove All Senders Refactoring: the call to this method in all senders is removed (as long as possible)
+I am a command to launch the Remove All Senders Refactoring: removes all possible senders from a method.
```

Comment:
The comment is still not clear for me, especially `removes all possible senders from a method.`
Maybe something like: `remove all message sends to this method` ?

Reply:
This comment was inconsistent with the refactoring it uses (the `RBRemoveAllSendersRefactoring`).
I have changed the comment so that it is the same as `RBRemoveAllSendersRefactoring`.
What it does is it removes all (that it possibly can) messages that are sent from the source code of this message.

Below is an example taken from `RBRemoveAllSendersRefactoring` class comment. 

Example Script
----------------
```smalltalk
| refactoring options |
refactoring := RBRemoveSenderRefactoring 
   remove: (90 to: 105) "node position to be removed "
   inMethod: #caller1
   forClass: RBRefactoryTestDataApp.
options := refactoring options copy.
options at: #inlineExpression put: [:ref :string | false].
refactoring options: options.
refactoring execute.
```
Before refactoring:
```smalltalk
RBRefactoryTestDataApp >> caller1
 | anObject |
 anObject := 5.
 self called: anObject + 1
  on1: 
   [:each | 
   each printString.
   ^anObject]
```
After refactoring (notice that the call to printstring was removed):
```smalltalk
RBRefactoryTestDataApp >> caller1
 | anObject |
 anObject := 5.
 self called: anObject + 1
  on1: 
   [:each | 
   ^anObject]
```

So both comments should be updated to reflect the usage.